### PR TITLE
New package: Obfuscatee v0.1.0

### PR DESCRIPTION
**Pull request by zsunberg**
UUID: 4b331413-18b5-5698-88d1-0abf3b5dfc34
Repo: https://github.com/sisl/Obfuscatee.jl
Tree: 1efb4c5b19688594d3bd4e13dd5f018eb1091138

Registrator tree SHA: 29a8496c5a7411ca37876fd237ef8a4a6f938ee0

### CODE REVIEW

**Comment by zsunberg** [comment]
@MaximeBouton , @rejuvyesh is it ok for me to register this? (I don't need a detailed review or anything)

**Comment by zsunberg** [comment]
oh wait! I think there is a mistake. let me try again

**Comment by rejuvyesh** [comment]
Why, this seemed fine to me? Version?

**Comment by zsunberg** [comment]
@rejuvyesh no I accidentally had deleted a dependency
